src/getHumanAge.test.js

```diff
@@ -7,4 +7,25 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should convert cat and dog ages correctly', () => {
+    expect(getHumanAge(23, 23)).toEqual([1, 1]);
+  });
+
+  test('should return 0 for ages less than the first threshould', () => {
+    expect(getHumanAge(10, 10)).toEqual([0, 0]);
+  });
+
+  test('should return 1 for ages between the first and second threshould',
+    () => {
+      expect(getHumanAge(17, 17)).toEqual([1, 1]);
+    });
+
+  test('should handle large age values correctly', () => {
+    expect(getHumanAge(100, 100)).toEqual([21, 17]);
+  });
+
+  test('should round down correctly for fractional years', () => {
+    expect(getHumanAge(27, 27)).toEqual([2, 2]);
```

Comment:
The expected result for 27 years is incorrect. For 27 cat years, the expected human age should be 3, and for 27 dog years, it should be 2. Update the expected result to `[3, 2]`.